plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -6354,6 +6354,53 @@ public void testCreateOrReplaceTableWithComments()
         }
     }
 
+    @Test
+    public void testCreateOrReplaceTableWithSameLocation()
+    {
+        try (TestTable table = new TestTable(
+                getQueryRunner()::execute,
+                "test_create_or_replace_with_same_location_",
+                " (a integer)")) {
```

Comment:
nit: No need to add a leading space as `TestTable` prepends it. Same for `testCreateOrReplaceTableWithChangeInLocation`.